src/vcpkg/dependencies.cpp

```diff
@@ -1895,13 +1944,62 @@ namespace vcpkg::Dependencies
                         "with detailed steps to reproduce the problem.");
                 }
 
+                const auto get_vars = [&]() {
```

Comment:
It seems like this would be better as just another member function (with a parameter `spec`) than as a lambda?

Reply:
Done